app/src/main/res/layout/fragment_reset_password.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/bg_authentication"
+    android:orientation="vertical">
+
+    <androidx.appcompat.widget.Toolbar
```

Comment:
This toolbar is empty. Did you forget to add a back button?
If so, then I suggest using an `ImageButton` instead, since you don't have a toolbar title